### PR TITLE
creating VM test

### PR DESCRIPTION
**Pull request by gonzalez1ucy**
Able to create a virtual machine and verify when it's created. Used various functions to alter the code and serve as a sub/reference for future cypress. Runs through the user scenario that would be done with cypress.

### CODE REVIEW

File: StartScreen/Javascript.js [hunk 1]
```diff
@@ -1,23 +1,40 @@
...
-document.getElementsByClassName("regionSelect")[1].click();	}
+document.getElementsByClassName("regionSelect")[1].click();	
+}
  region();
 function tabs(){
-document.getElementsByClassName("strong tab")[0].click();		}
+document.getElementsByClassName("strong tab")[0].click();		
+}
 tabs();
 function get(){
```
Comment (LuisYanchapanta) [inline StartScreen/Javascript.js]: really like the conjunction of the function to the task it is doing. The rest of them are well named function except I was just a little confused on get as function name, maybe OSBlock? TileofTab? not sure just that all of them seem well titled except this one.

File: StartScreen/Javascript.js [hunk 2]
```diff
@@ -1,23 +1,40 @@
...
+	return (true);
 }
+ready();
+ //if (ready === true ){setTimeout(function(){
+    window.location.href = 'https://www.paperspace.com/console/machines';
+    }, 120000);}
+    
+    
+    // setTimeout(function(){
+    window.location.href = 'https://www.paperspace.com/console/machines';
```
Comment (LuisYanchapanta) [inline StartScreen/Javascript.js]: I was a little confused why the function was repeated twice

**Review by LuisYanchapanta — COMMENTED** [review]
Code looks simple and to the point, really like the function titles. I was just had a few questions about line 31 to 36 cause I was just confused about why it was repeated. Overall it was done well, nice job!

File: StartScreen/Javascript.js (hunk 2 above)
Comment (gonzalez1ucy) [reply StartScreen/Javascript.js]: I think it was accidentally copied twice. I will make sure to delete it to make the code look cleaner.

File: StartScreen/Javascript.js (hunk 1 above)
Comment (gonzalez1ucy) [reply StartScreen/Javascript.js]: I'll change it to TileTabs just to make sure it is known what I am changing.

**Review by LuisYanchapanta — APPROVED** [review]
I like the edits and you made it more easier to read, Nice Work!

**Pull request merged**